docs/xmlprofiles.rst

```diff
@@ -1149,40 +1149,40 @@ The quality of service (QoS) handles the restrictions applied to the application
     :start-after: <!-->XML-QOS<-->
     :end-before: <!--><-->
 
-+--------------------------+----------------------------------+-------------------------+-------------------------+
-| Name                     | Description                      | Values                  | Default                 |
-+==========================+==================================+=========================+=========================+
-| ``<durability>``         | It is defined in                 |:class:`VOLATILE`,       | :class:`VOLATILE`       |
-|                          | :ref:`SettingDataDurability`     |:class:`TRANSIENT_LOCAL` |                         |
-|                          | section.                         |:class:`TRANSIENT`       |                         |
-|                          |                                  |                         |                         |
-+--------------------------+----------------------------------+-------------------------+-------------------------+
-| ``<liveliness>``         | Defines the liveliness of the    | :ref:`LivelinessType`   |                         |
```

Comment:
In the XML the liveliness tag still being defined following this example:

```xml
<liveliness>
    <kind>AUTOMATIC</kind> <!-- string -->
    <lease_duration>
         <sec>1</sec>
    </lease_duration>
    <announcement_period>
        <sec>1</sec>
    </announcement_period>
</liveliness>
```

But in your change, it appears only the kind as it was defined in the removed `LivelinessType` section. I think there is a loss of information here.

Reply:
My bad. I moved the `LivelinessType` section to the pub-sub section together with other QoS and explained the liveliness kind in the table instead of referring to the new section. It's fixed now.